README.md

```diff
@@ -1,6 +1,18 @@
-# Polka Native Storage
+# Polka Native Storage Research
+
+This repository contains the research and poc implementation of the Polka Native Storage grant project completed in Q4 2023. A Substrate-based system parachain intended to introduce decentralized storage technology to Polkadot.
+You can read the results [here](https://github.com/eigerco/polkadot-native-storage/blob/main/doc/report/polkadot-native-storage-v1.0.0.pdf)
```

Comment:
I think this small change will add more visibility to the core part of the PR

```suggestion
> [!IMPORTANT]
> This repository contains the research and poc implementation of the Polka Native Storage grant project completed in Q4 2023. A Substrate-based system parachain intended to introduce decentralized storage technology to Polkadot.
> You can read the results [here](https://github.com/eigerco/polkadot-native-storage/blob/main/doc/report/polkadot-native-storage-v1.0.0.pdf)
```

Renders like this: 
![image](https://github.com/user-attachments/assets/bcdd0c39-d6cb-4d71-a386-98893f6302e2)